implementation/test/DepositUtilsTest.js

```diff
@@ -376,7 +382,61 @@ describe("DepositUtils", async function() {
   })
 
   describe("auctionValue()", async () => {
-    it.skip("is TODO")
+    let duration
+    let basePercentage
+    before(async () => {
+      duration = await tbtcConstants.getAuctionDuration.call()
+      basePercentage = await tbtcConstants.getAuctionBasePercentage.call()
```

Comment:
Almost would be nice to make this `baseFactor` and do the division by 100 here, instead of having to do it manually in each test case.

Reply:
we do it twice, using different values. We also use `basePercentage` directly. Not sure I see why to change this